pages/wiki/creating-an-asteroid-app.md

```diff
@@ -152,10 +151,88 @@ Change the following `Environment` variables:
 
 Your app should now be able to run from the application when you click the start button in the bottom left sidebar.
 
+# Learn QML
+
+Main docs: https://doc.qt.io/ (AOS uses qt5)
+
+AOS specific component/API: https://github.com/AsteroidOS/qml-asteroid
+
+Examples of use of most of those can be found by searching the AsteroidOS codebase: https://github.com/search?q=org%3AAsteroidOS+StatusPage&type=code
+
+# Development Cycle
+
+Have 2 terminals:
+
+1. Do ssh and journalctl in one
+2. Other build app or vim
+
+QML Tester is an **on-watch** app to quickly test and debug qml's. You can install by running this command **on-watch**:
```

Comment:
debug QML files*
You can install it* by running

No need to use bold for "on-watch" the reader should get it ;)